### ISSUE TITLE
mini project 2 code review by He Li

### ISSUE THREAD

**Issue by helibu** [ruxinbu/ec601#2]
The implementation of the classification is fine. Since it's pretty easy to tell the difference between roses and sunflowers, the accuracy is good. But it seems a little bit overfitting from the loss plot. I think you could train with more pics to fix that.
